### ISSUE TITLE
zxfer resuming unneeded disks from sleep

### ISSUE THREAD

**Issue by petr-fischer** [allanjude/zxfer#26]
Hello, I have 3 ZFS pools on home server (zroot, zusbbackup, zmirror) and I am backuping FreeBSD system (zroot) to the USB drive (zusbbackup) with this cron job:

zxfer -dFPv -R zroot zusbbackup/backups/pf-server

I am putting my big media mirror (zmirror) to sleep after 40 minutes of inactivity.
But - when zxfer starts (zroot -> zusbbackup), my zmirror disks are resumed - I don't want to even touch the zmirror pool by zxfer while backuping different zpools.

Is possible, that zffer in some internal command call ZFS function, that acts over ALL zpools (some internal snapshot listing or something)?

Any idea?

Thanks very much!

**Comment by allanjude** [comment]
Yes, it does 'zfs list' across everything at the start of the replication

**Comment by cedrictineo** [comment]
@petr-fischer could you share how you put disks or pools to sleep? Thx

**Comment by petr-fischer** [comment]
@cedrictineo by setting timeout

`camcontrol idle ada1 -t 7200
camcontrol idle ada2 -t 7200
camcontrol idle da1 -t 2400`
